tools/run_tests/build_csharp_coreclr.sh

```diff
@@ -35,4 +35,4 @@ cd $(dirname $0)/../../src/csharp
 # TODO(jtattermusch): introduce caching
 dotnet restore .
 
-dotnet build -f netstandard1.5 --configuration $MSBUILD_CONFIG '**/project.json'
```

Comment:
Unfortunately, yes.   This command should be is "build all projects for coreCLR", that used to be simple because all coreCLR targets used to be netstandard1.5. Now some of them are netcoreapp1.0 (executables) and some of them are netstandard1.5 (libraries). And I didn't want to manually list all projects because that is error prone.